libs/utils-common/src/lib/data/index.ts

```diff
@@ -52,14 +49,12 @@ export function getAvgTestSuccess(uxTests: Partial<UxTest>[]) {
  * @param uxTests Array of tests associated to a page/task/project
  */
 export function getAvgSuccessFromLastTests(uxTests: Partial<UxTest>[]) {
-  //const lastTest = getLatestTest(uxTests);
 
-  // updated
   const uxTestsWithSuccessRate = uxTests
-  .filter((test) => test.success_rate);
-  //const lastTest = getLatestTest(uxTests.filter((test) => test.success_rate));
+  .filter((test) => test.success_rate && test.success_rate >= 0);
```

Comment:
You have to be really careful with comparison operators in javascript when working with `0` and optional properties because of how easy it is to accidentally cast things in a way that you didn't intend. `>= 0` is especially bad because if "falsy" values get cast to `0` for the numeric comparison, logically any value should be `>= 0`. But actually, `undefined` and `NaN` and the only two values that aren't `>= 0`.

That said, strict equality checks should always be preferred so I'll just change it to `success_rate ?? success_rate === 0`

Reply:
Thanks for your explanation Marc!